manifests/gke-storage.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "local-storage"
```

Comment:
Aliasing persistent disk as local-storage would confuse users. We can install TiDB Operator or TiDB cluster using `standard` storage class.

Reply:
I am trying to have this use local SSD, thus the `pd-ssd` type. I am following [these instructions](https://cloud.google.com/kubernetes-engine/docs/concepts/persistent-volumes#ssd_persistent_disks).

Reply:
SSD persistent disk is still network storage, not local SSD. So naming it `local-storage` confuses users.

Reply:
I agree. This makes the tutorial work right now though since it is what the current chart uses. Should we have the tutorial `helm install -set` to this with a different name?
I started a branch for local SSD that doesn't yet work called gregwebs/gke-local-ssd-setup

Reply:
Yes, we should have `helm install -set` for different storage class.

Reply:
We're going to release v0.1.0. This doesn't affect TiDB Operator's functionality, we can change it in another PR.